asyncapi/asb/Ballerina.toml

```diff
@@ -7,15 +7,15 @@ authors = ["Ballerina"]
 keywords = ["IT Operations/Message Brokers", "Cost/Paid", "Vendor/Microsoft"]
 icon = "docs/icon.png"
 repository = "https://github.com/ballerina-platform/asyncapi-triggers"
-distribution = "2202.4.1"
+distribution = "2201.11.0"
```

Comment:
Since we have the new ASB listener under the connector module itself, do we still need to keep this trigger?

Reply:
I think we have not yet deprecated this trigger and hence there could be usages. I think the proper approach should be mark them deprecated with U11, and remove them from U12. @daneshk @ThisaruGuruge highly appreciate your input here.

Reply:
As discussed offline we will disable `trigger.asb` and `trigger.salesforce` from the build and will do a separate release for those triggers for U11.